documentation/resource_spotlight.md

```diff
@@ -0,0 +1,87 @@
+spotlight
+===
+
+Use the **spotlight** resource to manage the metadata indexing state for disk volumes. This
+will primarily affect the ability to search volume contents with the macOS Spotlight feature.
+Under the hood, a **spotlight** resource executes the `mdutil` command in the `metadata_util`
+library.
+
+[Learn more about Spotlight](https://support.apple.com/en-us/HT204014).
+
+[Learn more about the `mdutil` command](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/mdutil.1.html).
+
+Syntax
+------
+
+The most basic usage of a **spotlight** resource block declares a disk volume as the name property
+to **enable** metadata indexing:
+
+```ruby
+spotlight '/'
+```
+
+The full syntax for all of the properties available to the **spotlight** resource is:
+
+```ruby
+spotlight 'volume name' do
+  volume                      String # defaults to 'volume name' if not specified
+  indexed                     TrueClass, FalseClass # defaults to TrueClass if not specified
+  searchable                TrueClass, FalseClass # defaults to TrueClass if not specified
+end
+```
+
+Actions
+-------
+
+This resource has the following actions:
+
+`:set`
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Set the metadata indexing state declared by the `indexed`
+property. This is the only, and default, action.
+
+
+Properties
+----------
+
+`volume`
```

Comment:
IMO I think this data would look more readable in plaintext with the use of a markdown table while still being quite digestible when rendered.